pkg/collector/volumeclaim.go

```diff
@@ -19,22 +19,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 
 	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
-	"github.com/open-telemetry/opentelemetry-operator/internal/config"
 )
 
 // VolumeClaimTemplates builds the volumeClaimTemplates for the given instance,
 // including the config map volume mount.
-func VolumeClaimTemplates(cfg config.Config, otelcol v1alpha1.OpenTelemetryCollector) []corev1.PersistentVolumeClaim {
-
-	var volumeClaimTemplates []corev1.PersistentVolumeClaim
-
+func VolumeClaimTemplates(otelcol v1alpha1.OpenTelemetryCollector) []corev1.PersistentVolumeClaim {
 	if otelcol.Spec.Mode != "statefulset" {
-		return volumeClaimTemplates
+		return []corev1.PersistentVolumeClaim{}
 	}
 
-	// Add all user specified claims or use default.
-	volumeClaimTemplates = append(volumeClaimTemplates,
-		otelcol.Spec.VolumeClaimTemplates...)
-
-	return volumeClaimTemplates
+	// Add all user specified claims.
+	return append([]corev1.PersistentVolumeClaim{}, otelcol.Spec.VolumeClaimTemplates...)
```

Comment:
Why not just return `otelcol.Spec.VolumeClaimTemplates`?

Reply:
Yes, why not 🤦 , thanks!